addons/mail/static/src/discuss/call/common/call_menu.js

```diff
@@ -9,6 +9,38 @@ export class CallMenu extends Component {
     setup() {
         super.setup();
         this.rtc = useState(useService("discuss.rtc"));
+        this.state = useState({
+            systrayIcon: undefined,
+        });
```

Comment:
Just `icon` rather than `systrayIcon`. "systray" leaks how the call menu is used (in systray), which won't necessarily be true in future, but also no reason to name a variable too specifically when there's no need for it.